src/components/MessageInput.tsx

```diff
@@ -0,0 +1,111 @@
+/** @jsxImportSource @emotion/react */
+import { useState, useRef, FormEvent } from 'react';
+import { css } from '@emotion/react';
+import Search from '/public/assets/search.png';
+import Send from '/public/assets/send.png';
```

Comment:
앞에 `/public` 경로는 명시하지 않아도 됩니다! 오히려 아래와 같은 경고가 뜨네요
```
Files in the public directory are served at the root path.
Instead of /public/assets/arrow-left.png, use /assets/arrow-left.png.
```